### ISSUE TITLE
the result of roberta-large

### ISSUE THREAD

**Issue by zzhinlp** [zjunlp/DocuNet#27]
Why was bert able to converge to 61 and Robert could only converge to about 47, using the original code and running on 3090

**Comment by zxlzr** [comment]
Hi, maybe you can try some different  hyper-parameters or re-run the code with more steps?

**Comment by zzhinlp** [comment]
Is there a processed CDR and GDA dataset？

**Comment by zzhinlp** [comment]
The reproduced results are normal on the validation set, but only 49 on the leaderboard test set.
What is the problem?

**Comment by njcx-ai** [comment]
Hi; generally, the results uploaded to the test will fluctuate by no more than one point. Yours is nearly 20 points lower. You can check whether there is a problem with the test data and code?
